src/components/views/verification/VerificationShowSas.js

```diff
@@ -45,7 +45,7 @@ export default class VerificationShowSas extends React.Component {
                 "Verify this user by confirming the following number appears on their screen.",
             )}</p>
             <p>{_t(
-                "For maximum security, we reccommend you do this in person or use another " +
+                "For maximum security, we recommend you do this in person or use another " +
```

Comment:
Good spot, although generally its better to open a separate PR for unrelated things like this (I know it seems small but the risk is it will get fixed elsewhere and then your PR conflicts).

Reply:
You're right again, sorry. This slipped through somehow, I did another PR that has already been accepted.